docs/source/terms_and_model.rst

```diff
@@ -346,6 +346,186 @@ Location for Variation.
+
+**Biological definition**
+
+Cytobands (cytological bands) are patterns observed on metahphase
+chromosomes after staining with dye. 
+
+**Computational definition**
+
+A CytobandLocation is a :ref:`Location` that is defined by a named
+chromosomal band within a species.  Conceptually, a CytobandLocation
+represents a family of :ref:`SequenceLocations` within a species.
+
+**Information model**
+
+.. list-table::
+   :class: reece-wrap
+   :header-rows: 1
+   :align: left
+   :widths: auto
+
+   * - Field
+     - Type
+     - Limits
+     - Description
+   * - _id
+     - :ref:`CURIE`
+     - 0..1
+     - Location Id; MUST be unique within document
+   * - type
+     - string
+     - 1..1
+     - Location type; MUST be set to **'CytobandLocation'**
+   * - species
+     - :ref:`CURIE`
+     - 1..1
+     - An external reference to a species taxonomy.  See Implementation Guidance, below.
+   * - chromsome
+     - string
+     - 1..1
+     - The symbolic chromosome name
+   * - start
+     - string
+     - 1..1
+     - The chromosmal band name of the start
+   * - end
+     - string
+     - 0..1
+     - The end chromosomal band name; if null, end is presumed to equal `start`.
+
+
+**Implementation guidance**
+
+* CytobandLocation is intended to enable the representation of
+  cytogenetic results from karyotyping or low-resolution molecular
+  methods.  Precise :ref:`SequenceLocations` should be preferred when
+  known.
+* `species` is specified using the NCBI taxonomy.  The CURIE prefix
+  MUST be `taxonomy`, corresponding to the `NCBI taxonomy prefix at
+  identifiers.org
+  <https://registry.identifiers.org/registry/taxonomy>`__.
+* Valid values for, and syntactic structure of, `chromosome`, `start`,
+  and `end` depend on the species.  Therefore, VRS does not constrain
+  these values in the model, except for the guidelines below.
+* `chromosome` MUST be an official primary name from `NCBI Assembly
+  <https://www.ncbi.nlm.nih.gov/assembly>`__.  For Humans, valid
+  chromosome names are 1..22, X, Y.
+* `start` and `end` SHOULD be values that are conventional for the
+  species. For Humans, bands are denoted by the arm (`p` or `q`) and
+  position (e.g., `22` or `22.3`). See example.
+* When `end` is provided, `CytobandLocation` is effectively a
+  contiguous interval of cytobands. There is no distinct class for
+  cytoband intervals.
+* Prescribing the conversion of CytobandLocations to SequenceLocations
+  is out-of-scope for VRS.  Recommended data for this operation are
+  available at `https://ftp.ncbi.nlm.nih.gov/pub/gdp/`__ and
+  `genome.ucsc.edu`__.
+
+
+**Example**
+
+.. parsed-literal::
+
+   {
+     'chr': '11',
+     'end': 'q22.3',
+     'species': 'taxonomy:9606',
+     'start': 'q22.2',
+     'type': 'CytobandLocation'
+   }
+
+
+.. _gene_location:
+
+GeneLocation
+$$$$$$$$$$$$$$$$
+
+**Biological definition**
+
+The location of a gene is the imprecise conceptual location of a gene
+feature on a named archetypal chromosome.
```

Comment:
I wrestled with whether GeneLocation had a biological definition at all. I started much the way you did (with a definition of a gene), but ultimately discarded it because defining gene seems out of scope. (After all, we're saying that we're going to delegate this definition to an external org, so by construction it is not ours to define.)

However, I can get behind this definition too. Sold!